src/averageAges.js

```diff
@@ -14,12 +14,25 @@
  *
  * @return {number}
  */
+const PERSON_SEX_MALE = 'm';
+const PERSON_SEX_FEMALE = 'f';
+const CENTURY_SEPARATOR = 100;
+
+function calculateAverageAge(filteredPeople) {
+  return filteredPeople.reduce(
+    (sum, { died, born }) => sum + (died - born),
+    0) / filteredPeople.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredPeople = century
+    ? people.filter(({ died }) =>
```

Comment:
<img width="1044" alt="image" src="https://github.com/mate-academy/js_average-ages/assets/86789161/daa2538d-7a5c-4aee-8bd2-d59d5c15b655">

Reply:
The same in all code